pages/contributing/metadata_guidelines.md

```diff
@@ -3,107 +3,145 @@ title: Metadata Guidelines
+  expose: true
+  type: institution
+  url: https://www.manchester.ac.uk/
+  pid: https://ror.org/027m9bs27
+```
 
-  ```yml
-  related_pages: 
-    - your_tasks: [page_id1, page_id2]
-    - your_domain: [page_id1, page_id2]
-  ``` 
+## Page metadata
+
+Each page in RSQKit can be described with the following metadata/attributes (which can either be included 
+in the "front matter" header of the page or applied to a group of pages via `_config.yml` repository file):
+
+* `title`: page title (used as the H1 HTML header in the rendered version of the page).
+* `summary`: a short page summary, displayed under the page title.
+* `description`: a short page description, used when pages are listed as "tiles" (e.g. in search results, related pages sections or when all pages of certain type are listed).
+* `type`: page type, used to classify pages into categories. Each page type is displayed differently in the website. Possible values: `research_community`, `task_page`.
+* `contributors`: a list of contributors that authored or contributed significantly to the page in some way (e.g. via discussions). Each contributor must also be listed in the `_data/CONTRIBUTORS.yml` repository file.
+* `search_exclude`: a boolean value indicating if the page should be excluded from search results. Default: `false`.
+* `sitemap`: a boolean value indicating if the page should appear in the `sitemap.xml`. Default: `true`.
+* `no_robots`: a boolean value indicating if the page should not end up in the search results of Google or any other search engines. Default: `false`.
+* `hide_sidebar`: a boolean value indicating if the page should not appear in the sidebar. Default: `false`.
+* `custom_editme`: specify an alternative file/link when clicking on the `edit-me` button.
+* `keywords`: a list of lowercase keywords that can be used to find the page using the search facility of RSQKit.
+* `sidebar`: name of the left-hand side navigation sidebar that should be displayed for the page. Default sidebar: `main`. The sidebar file `<SIDEBAR_NAME.md>` named after the sidebar must exist under `_data/sidebars/` in the repository.
+* `toc`: a boolean value indicating if a table of contents should be included at the top of the page. Default: `false`.
+* `page_id`: unique identifier of the page, usually a shortened version of the page title (with words separated with underscores or dashes and spaces avoided). This identifier is used in `related_pages` parameter to list pages related to this page. 
+* `datatable`: a boolean value indicating the activation of the pagination, sorting and searching in tabular representations of pages.
+* `related_pages`: a list of `page_id`s that are related to this page and will appear under "Related pages" section on the page, grouped by page type.
+* `page_citation`: When set to true, it will cause the citation section for the page to be generated in the format: "<author names>. <page title>. <site domain>. <page URL>. <date accessed>".
```

Comment:
What's the default - I thought it was true - also in the netlify I get format "...." rather than the text - maybe it's different if built locally